### PR TITLE
Add id/resource_quotas localization

### PR DESCRIPTION
**Pull request by phanama**
This PR addresses #15341, adding Indonesian translation for Resource Quota page.
/assign @girikuncoro 
/assign @wahyuoi

### CODE REVIEW

File: content/id/docs/concepts/policy/resource-quotas.md [hunk 1]
```diff
@@ -0,0 +1,624 @@
...
+
+Sebuah Resource Quota, didefinisikan oleh objek API `ResourceQuota`, menyediakan batasan-batasan
+yang membatasi konsumsi gabungan sumber daya komputasi untuk tiap Namespace. Resource Quota dapat
+membatasi jumlah objek yang dapat dibuat dalam sebuah Namespace berdasarkan tipenya, maupun jumlah
+seluruh sumber daya komputasi yang dapat dipakai oleh sumber daya API (misalnya Pod) di Namespace
+tersebut.
+
+Resource Quota bekerja sebagai berikut:
+
+- Tim-tim berbeda bekerja di Namespace yang berbeda pula. Saat ini cara pakai ini masih
```
[the remark this answers is not in the document]
Comment (phanama) [reply content/id/docs/concepts/policy/resource-quotas.md]: I think this refers to the per-team-namespaces usage pattern generally (pola pakai), not directly to the usage of Resource Quota.
But I'll change the wording for clarity. Thanks!

**Comment by phanama** [comment]
I've pushed the fixes, pls review! @wahyuoi

File: content/id/docs/concepts/policy/resource-quotas.md [hunk 2]
```diff
@@ -0,0 +1,622 @@
...
+
+Sebuah Resource Quota, didefinisikan oleh objek API `ResourceQuota`, menyediakan batasan-batasan
+yang membatasi konsumsi gabungan sumber daya komputasi untuk tiap Namespace. Resource Quota dapat
+membatasi jumlah objek yang dapat dibuat dalam sebuah Namespace berdasarkan tipenya, maupun jumlah
+seluruh sumber daya komputasi yang dapat dipakai oleh sumber daya API (misalnya Pod) di Namespace
+tersebut.
+
+Resource Quota bekerja sebagai berikut:
+
+- Tim-tim berbeda bekerja pada Namespace yang berbeda pula. Saat ini hal ini belum diwajibkan,
```
Comment (wahyuoi) [inline content/id/docs/concepts/policy/resource-quotas.md]:
```suggestion
- Tim-tim berbeda bekerja pada Namespace yang berbeda pula. Sekarang hal ini belum diwajibkan,
```
how about using _sekarang_, so there will be one _ini_ word?

Comment (phanama) [reply content/id/docs/concepts/policy/resource-quotas.md]: Agreed. Done!

Comment (phanama) [reply content/id/docs/concepts/policy/resource-quotas.md]: @wahyuoi

**Pull request merged**